packages/plugin-data/src/dataCommand.ts

```diff
@@ -94,7 +94,7 @@ export abstract class DataCommand extends SfdxCommand {
   }
 
   public getConnection(): BaseConnection {
-    const connection: BaseConnection & ConnectionInternals = this.flags.useToolingApi
+    const connection: BaseConnection & ConnectionInternals = this.flags.usetoolingapi
```

Comment:
do all commands that have that flag extend `dataCommand` and use this `getConnection` method?

Reply:
@WillieRuemmele create/update/delete/get/query use the flag usertoolingapi

Reply:
yes, they do.

Reply:
yea, but a few weeks ago `soql:query` didn't extend `DataCommand` and used `this.org.getConnection` not `this.getConnection` I just want to make sure this code is being called for all commands that could use it